Develop

NadieinOleh opened the pull request:
https://nadieinoleh.github.io/layout_html-form/
https://nadieinoleh.github.io/layout_html-form/report/html_report/

vkryvytskyy commented on src/style.css:
```diff
@@ -0,0 +1,8 @@
+  margin-bottom: 20px;
+}
+
+.inputs-fieds {
```
```suggestion
.inputs-fields {
```

vkryvytskyy commented on src/index.html:
```diff
@@ -9,7 +9,63 @@
+    <fieldset class="blocks">
+      <legend>Personal information</legend>
+      <label for="surname">Surname:</label>
+      <input type="text" id="surname" name="surname" class="inputs-fieds" autocomplete="off"><br>
```
```suggestion
      <input type="text" id="surname" name="surname" class="inputs-fieds" autocomplete="off">
```
instead of using br, you could add **display: block** property directly to label, and put related input inside it. It gives you possibility to omit **for** and **id** attributes

vkryvytskyy commented on src/index.html:
```diff
@@ -9,7 +9,63 @@
+        <label for="yes">Yes</label>
+        <input type="radio" id="no" name="catsproblem" value="no">
+        <label for="no">No</label>
+        </label><br>
```
Inputs with type checkbox/radio is better to wrap by labels.
![image](https://user-images.githubusercontent.com/38065883/75265503-aa11fc00-57f9-11ea-85f6-53a730518c90.png)
![image](https://user-images.githubusercontent.com/38065883/75265514-ac745600-57f9-11ea-8eca-586e4d73e2a9.png)

vkryvytskyy commented on src/index.html:
```diff
@@ -9,7 +9,62 @@
+    <fieldset class="blocks">
+      <legend>An interesting fact about you!</legend>
+      <label class="inputs-fields" for="yes">Do you love cats?</label>
+      <label class="radio-inputs"><input type="radio" id="yes" name="catsproblem" value="yes">
```
One tag - one line. Use new lines for nested tags. Don't write nested tags in one line.

NadieinOleh committed:
Realise task html form

NadieinOleh committed:
Realise task html form

NadieinOleh committed:
Change in the readme

NadieinOleh committed:
Realise task html form

agniya-i commented on src/index.html:
```diff
@@ -11,7 +11,98 @@
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"       method="post">
+      <fieldset class="wrapper" >
```
```suggestion
      <fieldset class="wrapper">
```

agniya-i commented on src/index.html:
```diff
@@ -11,7 +11,98 @@
+        <legend>Personal information</legend>
+        <label class="inputs">
+          Surname:
+          <input type="text"
```
```suggestion
          <input 
            type="text"
```
start first attribute on a new line (checklist p3), please fix this everywhere

agniya-i commented on src/index.html:
```diff
@@ -11,7 +11,98 @@
+        </label>
+        <label class="inputs">
+          Name:
+          <input type="text"  autocomplete="off">
```
```suggestion
          <input type="text" autocomplete="off">
```

agniya-i commented on src/index.html:
```diff
@@ -11,7 +11,98 @@
+            min="1"
+            max="100"
+            required
+          >
```
```suggestion
          <input 
            type="number" 
            name="old"
            value="12"
            min="1"
            max="100"
            required
          >
```

agniya-i commented on src/index.html:
```diff
@@ -11,7 +11,98 @@
+        </label>
+        <label class="inputs">
+          I accept the term of the agreement
+          <input type="checkbox"  required>
```
```suggestion
          <input type="checkbox" required>
```
fix everywhere

agniya-i commented on src/index.html:
```diff
@@ -11,7 +11,98 @@
+        <legend>Additional info:</legend>
+        <label class="inputs">Comments:
+          <textarea   cols="10" rows="2"></textarea>
+        </label>
```
```suggestion
        <label class="inputs">
          Comments:
          <textarea cols="10" rows="2"></textarea>
        </label>
```

agniya-i commented on src/index.html:
```diff
@@ -11,7 +11,98 @@
+          </select>
+        </label>
+      </fieldset>
+      <button type="submit">Submit </button>
```
```suggestion
      <button type="submit">Submit</button>
```

agniya-i commented on src/index.html:
```diff
@@ -11,7 +11,98 @@
+        <legend> Registration</legend>
+        <label class="inputs">
+          E-mail:
+          <input type="email"  placeholder="email@example.com" >
```
every field should have `name` attribute

agniya-i reviewed (changes requested):
Don't forget to add demo links to commit message

NadieinOleh committed:
fixed what the mentor wrote

NadieinOleh commented:
https://nadieinoleh.github.io/layout_html-form/

sTorba24 commented on src/index.html:
```diff
@@ -11,7 +11,131 @@
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"    method="post"
+    >
```
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
    >
```

sTorba24 replied:
move attributes to new line with indentation

sTorba24 commented on src/index.html:
```diff
@@ -11,7 +11,131 @@
+    >
+      <fieldset class="wrapper">
+        <legend>Personal information</legend>
+        <label class="inputs">
```
this is like item of fieldset, i think you can name using this informations. `inputs` doesn't seem like good name

sTorba24 commented on src/index.html:
```diff
@@ -11,7 +11,131 @@
+          >
+        </label>
+      </fieldset>
+      <fieldset class="wrapper">
```
use spaces between siblings
fix it everywhere please

sTorba24 reviewed (changes requested):
Good work! Some comments to improve code

NadieinOleh committed:
change classes and spaces between siblings

anstsot commented on src/index.html:
```diff
@@ -11,7 +11,136 @@
+            maxlength="8"
+          >
+        </label>
+        <label class="fields">
```
would be great to add an empty line between siblings

anstsot replied:
please check other places as well

anstsot reviewed (commented):
Would be great to add empty lines between siblings for better readability

NadieinOleh committed:
add spaces beetween siblings

polosanya commented on src/index.html:
```diff
@@ -11,7 +11,148 @@
+
+        <label class="fields">
+          Name:
+          <input
+          type="text"
+          name="userName"
+          autocomplete="off"
+          >
```
```suggestion
          <input
            type="text"
            name="userName"
            autocomplete="off"
          >
```
![image](https://user-images.githubusercontent.com/96409563/220445719-b9109521-27c4-4e1c-93c8-c7cee9fbef9c.png)

NadieinOleh replied:
Tag’s closing bracket   (closing-and-opening-   
                should be on the same   on-seme-level)          
                level as opening one
         <textarea
            name="feedBack"
            cols="10"
            rows="2"
          ></textarea>

polosanya commented on src/index.html:
```diff
@@ -11,7 +11,148 @@
+            name="feedBack"
+            cols="10"
+            rows="2">
+          </textarea>
```
```suggestion
          <textarea
            name="feedBack"
            cols="10"
            rows="2"
          ></textarea>
```
Let's fix this problem
![image](https://user-images.githubusercontent.com/96409563/220445863-b9d6ad4a-558b-444c-bb1a-007ad4c598ea.png)
![image](https://user-images.githubusercontent.com/96409563/220445916-bfa60bc4-0c3f-4852-a3b6-06c217fab8fe.png)

polosanya reviewed (changes requested):
Almost done! There is no need to close\reopen the PR
![image](https://user-images.githubusercontent.com/96409563/220446058-b10fb63d-0043-4dde-981c-827203956f5d.png)
Just push the changes and re-request a review

NadieinOleh committed:
Update src/index.html

Tag’s closing bracket   (closing-and-opening-   
                should be on the same   on-seme-level)          
                level as opening one

Co-authored-by: Oleksandr Polotniuk <96409563+polosanya@users.noreply.github.com>

qoa11a commented on src/index.html:
```diff
@@ -11,7 +11,148 @@
+
+        <label class="fields">
+          Name:
+          <input
+          type="text"
+          name="userName"
+          autocomplete="off"
+          >
```
```suggestion
          <input
            type="text"
            name="userName"
            autocomplete="off"
          >
```
yeah, opening and closing brackets should be at the same level, you're right
but if element has 2 attributes or more - start each one, including the first, on the new line with 2-space indentation related to tag

NadieinOleh committed:
Update

https://nadieinoleh.github.io/layout_html-form/
https://nadieinoleh.github.io/layout_html-form/report/html_report/

sTorba24 commented on src/index.html:
```diff
@@ -11,7 +11,148 @@
+
+        <label class="fields">
+          Name:
+          <input
+          type="text"
+          name="userName"
+          autocomplete="off"
+          >
```
```suggestion
          <input
            type="text"
            name="userName"
            autocomplete="off"
          >
```

sTorba24 replied:
Fix it as asked before ;)

sTorba24 commented on src/index.html:
```diff
@@ -11,7 +11,148 @@
+
+        <label class="fields">
+          Comments:
+          <textarea
+            name="feedBack"
+            cols="10"
+            rows="2"
+         ></textarea>
```
```suggestion
          <textarea
            name="feedBack"
            cols="10"
            rows="2"
           ></textarea>
```

sTorba24 reviewed (approved):
Approved! But look at the CR comments more properly to not forget fix something :)
